docs/trulens_eval/intro.md

```diff
@@ -6,58 +6,56 @@ Evaluate and track your LLM experiments with TruLens. As you work on your models
+    * Anything that is tracked by the instrumentation can be evaluated!
+
+The process for building your evaluated and tracked LLM application with TruLens is below 👇
+
 ![Architecture Diagram](https://www.trulens.org/Assets/image/TruLens_Architecture.png)
 
+## Installation and Setup
+
+Install the trulens-eval pip package from PyPI.
+
+```bash
+    pip install trulens-eval
+```
+
 ## Quick Usage
 
-To quickly play around with the TruLens Eval library:
+TruLens supports the evaluation of tracking for any LLM app framework. Choose a framework below to get started:
 
-Langchain:
+**Langchain**
 
 [langchain_quickstart.ipynb](https://github.com/truera/trulens/blob/releases/rc-trulens-eval-0.7.0/trulens_eval/examples/quickstart.ipynb).
 [![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/truera/trulens/blob/releases/rc-trulens-eval-0.7.0/trulens_eval/examples/colab/quickstarts/langchain_quickstart_colab.ipynb)
 
 [langchain_quickstart.py](https://github.com/truera/trulens/blob/releases/rc-trulens-eval-0.7.0/trulens_eval/examples/quickstart.py).
 
-Llama Index: 
+**Llama-Index**
 
 [llama_index_quickstart.ipynb](https://github.com/truera/trulens/blob/releases/rc-trulens-eval-0.7.0/trulens_eval/examples/frameworks/llama_index/llama_index_quickstart.ipynb).
 [![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/truera/trulens/blob/releases/rc-trulens-eval-0.7.0/trulens_eval/examples/colab/quickstarts/llama_index_quickstart_colab.ipynb)
 
 [llama_index_quickstart.py](https://github.com/truera/trulens/blob/releases/rc-trulens-eval-0.7.0/trulens_eval/examples/llama_index_quickstart.py)
 
-No Framework: 
+**No Framework**
 
 [no_framework_quickstart.ipynb](https://github.com/truera/trulens/blob/releases/rc-trulens-eval-0.7.0/trulens_eval/examples/no_framework_quickstart.ipynb).
 [![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/truera/trulens/blob/releases/rc-trulens-eval-0.7.0/trulens_eval/examples/colab/quickstarts/no_framework_quickstart_colab.ipynb)
 
 [no_framework_quickstart.py](https://github.com/truera/trulens/blob/releases/rc-trulens-eval-0.7.0/trulens_eval/examples/no_framework_quickstart.py)
 
-### 💡 Contributing
-
-Interested in contributing? See our [contribution guide](https://github.com/truera/trulens/tree/main/trulens_eval/CONTRIBUTING.md) for more details.
-
-## Installation and Setup
-
-Install the trulens-eval pip package from PyPI.
-
-```bash
-    pip install trulens-eval
-```
-
-### API Keys
 
-Our example chat app and feedback functions call external APIs such as OpenAI or HuggingFace. You can add keys by setting the environment variables. 
-
-#### In Python
-
-```python
-import os
-os.environ["OPENAI_API_KEY"] = "..."
-```
-
-#### In Terminal
```

Comment:
Why do we remove this? Isn't this the key instructions as to how to get started?

Reply:
Replaced the code with links to 3 different quickstarts to emphasize TruLens is cross-framework.